DeusEx/Classes/DXREntranceRando.uc

```diff
@@ -522,10 +522,99 @@ function RandoMission6()
     GenerateConnections(6);
 }
 
+function RandoMission8()
+{
+    AddDoubleXfer("08_NYC_BAR","ToBarFrontEntrance","08_NYC_Street","FromBarFrontEntrance");
+    AddDoubleXfer("08_NYC_BAR","ToBarBackEntrance","08_NYC_Street","FromBarBackEntrance");
+    AddDoubleXfer("08_NYC_FREECLINIC","FromStreet","08_NYC_Street","FromClinic");
+    AddDoubleXfer("08_NYC_HOTEL","ToHotelFrontDoor","08_NYC_Street","FromHotelFrontDoor");
+    AddDoubleXfer("08_NYC_HOTEL","ToHotelBedroom","08_NYC_Street","BedroomWindow");
+    AddDoubleXfer("08_NYC_SMUG","ToSmugFrontDoor","08_NYC_Street","FromSmugFrontDoor");
+    AddDoubleXfer("08_NYC_SMUG","ToSmugBackDoor","08_NYC_Street","FromSmugBackDoor");
+    AddDoubleXfer("08_NYC_UNDERGROUND","ToNYCSump","08_NYC_Street","FromNYCSump");
+    AddDoubleXfer("08_NYC_UNDERGROUND","ToNYCUndergroundSewer2","08_NYC_Street","FromNYCUndergroundSewer2");
+
+    GenerateConnections(8);
+}
+
+function RandoMission9()
+{
+    AddDoubleXfer("09_NYC_DOCKYARD","ToDockyardSewer","09_NYC_Ship","FromDockyardSewer");
+    AddDoubleXfer("09_NYC_DOCKYARD","FromAircondDuct","09_NYC_ShipFan","ToAircondDuct");
+    AddDoubleXfer("09_NYC_DOCKYARD","ExitShip","09_NYC_Ship","EnterShip");
+    //AddDoubleXfer("09_NYC_DOCKYARD","","09_NYC_Graveyard","");
+    //AddDoubleXfer("09_NYC_GRAVEYARD","","10_Paris_Catacombs","");
+    AddDoubleXfer("09_NYC_SHIP","FromFanRoom","09_NYC_ShipFan","ToFanRoom");
+    AddDoubleXfer("09_NYC_SHIP","FromDuctRoom","09_NYC_ShipFan","ToDuctRoom");
+    AddDoubleXfer("09_NYC_SHIP","FromBelow","09_NYC_ShipBelow","FromAbove");
+
+    GenerateConnections(9);
+}
+
+function RandoMission10()
+{
+    AddDoubleXfer("10_PARIS_CATACOMBS","spiralstair","10_Paris_Catacombs_Tunnels","spiralstair");//same tag on both sides?
+    //AddDoubleXfer("10_PARIS_CATACOMBS_TUNNELS","","10_Paris_Metro","sewer");//one way?
+    AddDoubleXfer("10_PARIS_CHATEAU","Chateau_start","11_Paris_Cathedral","cathedralstart");
+    AddDoubleXfer("10_PARIS_CLUB","Paris_Club1","10_Paris_Metro","Paris_Metro1");
+    AddDoubleXfer("10_PARIS_CLUB","Paris_Club2","10_Paris_Metro","Paris_Metro2");
+    //AddDoubleXfer("10_PARIS_METRO","sewer","");
+    //AddDoubleXfer("10_PARIS_METRO","","10_Paris_Chateau");
+
+    //AddDoubleXfer("11_PARIS_CATHEDRAL","cathedralstart","");
+    AddDoubleXfer("11_PARIS_CATHEDRAL","Paris_Underground","11_Paris_Underground","Paris_Underground");
+    //AddDoubleXfer("11_PARIS_EVERETT","","12_Vandenberg_cmd");
```

Comment:
mixing mission 10 and 11 didn't seem to have any adverse side-effects, but randomized entrances for Paris are broken in general